docs/monitoring.md

```diff
@@ -289,7 +289,7 @@ API path listed as below:
 
 | Path                  | Meaning                                                                                                     |
 |-----------------------|-------------------------------------------------------------------------------------------------------------|
-| /metrics/prometheus   | List the metrics data in prometheus format of the master.                                                   |
+| /metrics/prometheus   | List the metrics data in prometheus format of the master.(Configure `celeborn.metrics.prometheus.path`)     |
```

Comment:
List the metrics data in prometheus format of the master.(The url path is defined by configure `celeborn.metrics.prometheus.path`.)